kart/workdir.py

```diff
@@ -380,6 +384,14 @@ def reset(
             self.write_full_datasets_to_workdir(
                 [target_datasets[d] for d in ds_inserts]
             )
+        # Update the working copy with files that have changed:
+        kart_attachments = os.environ.get("KART_ATTACHMENTS")
+        if kart_attachments == "true":
```

Comment:
Previously we've just checked if the value of these env-variables is truthy by python standards - this means that "True" or "YES" or "1" will also work.
It does mean that "0" or "False" don't disable this properly but generally people will disable this by completely unsetting the variable, so it works pretty well for how simple it is.